libkbfs/quota_usage.go

```diff
@@ -118,7 +118,7 @@ func (q *EventuallyConsistentQuotaUsage) Get(
 	}()
 	past := q.config.Clock().Now().Sub(c.timestamp)
 	switch {
-	case past > blockTolerance:
+	case past > blockTolerance || c.timestamp.IsZero():
```

Comment:
When `c.timestamp` is zero value, shouldn't `past > blockTolerance` be `true` almost for sure?